player.py

```diff
@@ -267,7 +275,7 @@ def take_food_from_trail(self, trail, ant_positions, ant, direction):
 
     return (food_to_hand, trail)
 
-  def move_ant(self, trail, ant_positions, anthill, ant):
+  def move_ant(self, trail, ant_positions, anthill, anthill_order, ant):
```

Comment:
Where is this function every used? I don't think it's every used because of the way that `take_turn()` was developed. Therefore, I suggest you delete this whole function and all its tests. Keeping a function that is never used adds unnecessary overhead to the maintenance of the code.